### DISCUSSION TITLE
Is there a way to use esm.sh with local node packages (npm/yarn link)

### DISCUSSION THREAD

**Discussion by jacque006** [esm-dev/esm.sh#216] [Q&A]
We have a [monorepo](https://github.com/jzaki/bls-wallet) with a variety of components/subfolders in it, most of which are npm/NodeJS packages. However, one is a Deno process (`aggregator`) which brings a variety of npm dependencies in via esm.sh. One of them is our `clients` subfolder (`bls-wallet-clients`) https://github.com/jzaki/bls-wallet/blob/main/aggregator/deps.ts#L41-L45

When updating `clients` code and checking that everything works in a local dev env, I can use `npm/yarn link` to use my local changes to that package:
```sh
cd ./clients
yarn link bls-wallet-clients
cd ../extension
yarn link bls-wallet-clients
```

However with `aggregator` (Deno) I have been unable to find a similar way I can bring in local changes to `clients` (`bls-wallet-clients`) with esm.sh or another tool. Is there some way I could accomplish this?

**Comment by ije** [comment]
did you try the import maps?

**Reply by jacque006** [reply]
I have not. Assuming you mean https://deno.land/manual/linking_to_external_code/import_maps/ , I don't see how that would allow me to do local package linking.

**Reply by ije** [reply]
you mean the local npm package？
